nipype/algorithms/misc.py

```diff
@@ -245,7 +245,7 @@ def _run_interface(self, runtime):
         else:
             affine = None
 
-        with open(self.inputs.header_file, 'rb') as data_file:
+        with open(self.inputs.data_file, 'rb') as data_file:
```

Comment:
this interface seems to only work with `hdr/img` pair files. in that space, shouldn't this be the `header_file`?